scripts/SumBlockForOne.ts

```diff
@@ -0,0 +1,68 @@
+// Run: npx hardhat run scripts/SumBlockForOne.ts --network childTest
+import { getTransactionsByBlock, decodeTransaction } from "./_helper";
+
+// Input parameters for the script:
+const BLOCK_NUMBER = 1301000;
+const MAX_BLOCK_NUMBER = 1335500;
+const VALIDATOR_ADDRESS = process.env.ADDRESS_FOR_SCRIPTS;
+const CONTRACT_ADDRESS = "0x0000000000000000000000000000000000000105";
```

Comment:
Which contract this address refers to? Is it for the ValidatorSet, RewardPool or Faucet?

Reply:
RewardPool, added comments, so it will be more clear